SUPPORT.md

```diff
@@ -0,0 +1,49 @@
+# Support for the Toit framework
+
+Welcome to Toit!
+
+# Documentation
+
+The [Toit docs](https://docs.toit.io) have lots of information. Some of it is
```

Comment:
I think the distinction between 'Toit docs' and toitdocs here is confusing. Maybe change this one to 'Toit documentation'?

Reply:
done.